build/source/engine/canopySnow.f90

```diff
@@ -96,10 +98,16 @@ subroutine canopySnow(&
  real(dp)                      :: throughfallDeriv           ! derivative in throughfall flux w.r.t. canopy storage (s-1)
  real(dp)                      :: unloadingDeriv             ! derivative in unloading flux w.r.t. canopy storage (s-1)
  real(dp)                      :: scalarCanopyIceIter        ! trial value for mass of ice on the vegetation canopy (kg m-2) (kg m-2)
+ real(dp)                      :: accum
```

Comment:
add comment that explains what `accum` is. Also consider using a more informative variable name (and carry it throughout)

Reply:
Removed this, it was not used anywhere else.